build.gradle

```diff
@@ -151,6 +151,7 @@ rat {
   excludes = exclusions
 }
 check.dependsOn rat
+check.dependsOn ":beam-sdks-java-javadoc:allJavadoc"
```

Comment:
This means that the root project will check that all javadocs are well formed which is typically something that we don't want to impose when running `check` within the root project.

I think you meant to edit BeamModulePlugin.groovy and add:
`project.check.dependsOn project.javadoc` within `applyJavaNature`

Reply:
Done, moved to BeamModulePlugin.groovy